### ISSUE TITLE
Program crash while viewing categorys

### ISSUE THREAD

**Issue by hegerdes** [s-h-3-l-l/katoolin3#6]
**Crash of python-script**
The script crashes while viewing categorys. This happens because of accessing an invalid dictionary key in apt.cache

**To Reproduce**
1. Start the script as sudo
2. Type 0 for categorys
3. Type 5 for passowrdcracking
4. Crash

**Expected behavior**
Show a list of tools

**Desktop:**
 - OS: Debian 10 (Buster) Kernel 4.19.06
 - katoolin3 (pulled it today from github)
 - python 3.7

**Additional context**
```bash
#!/bin/bash
5
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/apt/cache.py", line 293, in __getitem__
    rawpkg = self._cache[key]
KeyError: 'keimpx'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/katoolin3", line 1094, in <module>
    main()
  File "/usr/local/bin/katoolin3", line 1036, in main
    choice()
  File "/usr/local/bin/katoolin3", line 981, in view_categories
    view_packages(choice)
  File "/usr/local/bin/katoolin3", line 923, in view_packages
    if APT[pkg].is_installed:
  File "/usr/local/bin/katoolin3", line 657, in __getitem__
    return self._cache[item]
  File "/usr/lib/python3/dist-packages/apt/cache.py", line 295, in __getitem__
    raise KeyError('The cache has no package named %r' % key)
KeyError: "The cache has no package named 'keimpx'"
```

**Comment by s-h-3-l-l** [comment]
@hegerdes thanks for the issue,

I forgot to catch the KeyError exception there.
Please update your repo clone with [update.sh](https://github.com/s-h-3-l-l/katoolin3/blob/master/update.sh) and then it should work.

This was fixed in commit 428f78df109201e4e122bdb9e9685b4e33ebd66a.

**Comment by hegerdes** [comment]
Thanks! I know its hard to think about all possible errors. Maybe some tests for the future would be nice.

Other than that everything seems fine. Only a small formatting hickup in category 5 but nothing serious.
